zer0-os/zOS
# feat(message): set placeholder dimensions and ensure placeholders match aspect ratio of loaded images

### What does this do?
- Update placeholder to take width and height of the media file and set styles
- Ensures that image placeholders maintain the same aspect ratio as the loaded images to prevent layout jolting.

### Why are we making this change?
- We are making this change to improve the user experience by preventing layout shifts and ensuring that the placeholders and loaded images align correctly.

### How do I test this?
- Run tests as usual.
- Run the UI, send image in conversations, refresh page and check behaviour as per demo below.

### Key decisions and Risk Assessment:
  #### Things to consider:
  1. How will this affect security?
  1. How will this affect performance?
  1. Does this change any APIs?




https://github.com/user-attachments/assets/ed595b64-f405-464b-af45-040ec7fadd2d
